src/lsdb/example_module.py

```diff
@@ -0,0 +1,23 @@
+"""An example module containing simplistic functions."""
+
+
+def greetings() -> str:
+    """A friendly greeting for a future friend.
+
+    Returns
+    -------
+    str
+        A typical greeting from a software engineer.
+    """
+    return "Hello from LINCC-Frameworks!"
+
+
+def meaning() -> int:
+    """The meaning of life, the universe, and everything.
+
+    Returns
+    -------
+    int
+        The meaning of life.
+    """
+    return 42
```

Comment:
Do we need this?

Reply:
This is useful to have *something* tested in the workflows, even if it's meaningless.

I think I'd prefer something obviously meaningless to something that looks like it might be right but isn't (*cough* ChatGPT *cough*).

Reply:
Haha, fair enough